pkg/controller/deployment/cert_manager_cainjector_deployment.go

```diff
@@ -0,0 +1,58 @@
+package deployment
+
+import (
+	"github.com/openshift/cert-manager-operator/pkg/operator/assets"
+	"github.com/openshift/cert-manager-operator/pkg/operator/kubeclient"
+	configv1 "github.com/openshift/client-go/config/clientset/versioned/typed/config/v1"
+	"github.com/openshift/library-go/pkg/controller/factory"
+	"github.com/openshift/library-go/pkg/operator/events"
+	"github.com/openshift/library-go/pkg/operator/staticresourcecontroller"
+	"github.com/openshift/library-go/pkg/operator/status"
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
+	"k8s.io/client-go/informers"
+)
+
+const (
+	certManagerCAInjectorStaticResourcesControllerName = operatorName + "-cainjector-static-resources-"
+	certManagerCAInjectorDeploymentControllerName      = operatorName + "-cainjector-deployment-"
+	certManagerCAInjectorDeploymentFile                = "cert-manager-deployment/cert-manager-cainjector-deployment.yaml"
+)
+
+var (
+	certManagerCAInjectorAssetFiles = []string{
+		"cert-manager-deployment/cert-manager-cainjector-crb.yaml",
+		"cert-manager-deployment/cert-manager-cainjector-sa.yaml",
+		"cert-manager-deployment/cert-manager-cainjector:leaderelection-rb.yaml",
```

Comment:
don't use colons in filenames

organize the dirtree so that each of these controllers gets its own directory

Reply:
Done.